src/main/java/org/idpass/lite/IDPassLite.java

```diff
@@ -0,0 +1,17 @@
+package org.idpass.lite;
+
+/**
+ * Initializer for standard Java applications.
+ */
+public class IDPassLite {
```

Comment:
@typelogic may I ask why there are two different classes with same name for `IDPassLite`?

Reply:
To factor out JNI initialization and avoid `java.nio`. A separate `IDPassLite` handles one fore Android and for Java. If `JNI` initialization is within `IDPassReader` it leaks out `java.nio` to Android

Reply:
@typelogic ahh I understand. Just to confirm will this affect if for example smartscanner uses the android maven? Will we have access or the on

> > ahh okay understood @typelogic if that is the case. Just to confirm will it affect if for example our smartscanner app uses maven android repo. Will `IDPassLite` for java be visible/usable there or only for android? and also for java, vice versa?
> 
> Hi @rjmangubat23 , in `build.gradle` of the android project there is an `exclude` directive to exclude the Java version of `IDPassLite.java` at compile time. Therefore, only the Android version of `IDPassLite.java` gets pulled into the `aar`. If unzipping the `jar` or `aar` it will be seen that only one file `IDPassLite.class` exists. I named theme similar to simplify surface API which is `IDPassLite::initialize` for both.

okay @typelogic noted. I will approve this then. Thank you for clarifying!